### DISCUSSION TITLE
Welcome and rules!

### DISCUSSION THREAD

**Discussion by K4thos** [ikemen-engine/Ikemen-GO#944] [Announcements]
## 👋 Welcome!
We are using Discussions as a place for idea engineering with other members of our community. We hope that you:
* Share ideas.
* Comment and vote on feature requests.
* Engage with other community members.

Fulfilled ideas will be converted to show up on the [issues page](https://github.com/ikemen-engine/Ikemen-GO/issues) and archived as completed there.

Q&As are allowed here, but in most cases we prefer that you use the *ikemen-help* section of our **Discord** server, which you can find an invitation to on the [Ikemen GO website](https://ikemen-engine.github.io/). This is because many problems can be solved by members of the community who do not use GitHub. Please remember to check the [wiki](https://github.com/ikemen-engine/Ikemen-GO/wiki) page before asking a question and use the search bar before creating a new feature request topic. Bug reports and compatibility issues with mugen content should be posted on the issues page.

**Comment by LasombraDemon** [comment]
Shouldn't this one be pinned to the top? :P

**Comment by TIGRE2410** [comment]
Hi, I would like ikemen to be able to use the scalefx-aa shader from retroarch, it's the best shader I've seen...
i think scalefx is .glsl and ikemen only supports .frag and .vert in external/shaders folder

**Reply by Lazin3ss** [reply]
you should open a thread asking for this feature in "Feature Request" category

**Reply by TIGRE2410** [reply]
where is the feature request category?

**Comment by saefullohahmad71-alt** [comment]
Please fix the application that can't be opened, as soon as it's opened it immediately exits by itself

https://github.com/user-attachments/assets/02d6f640-fd1d-4672-b494-4cbbb25f7cbb
